src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -135,14 +137,17 @@ renderHsModuleForDotProto extraInstanceFiles dotProto importCtxt = do
 -- Instances given in @eis@ override those otherwise generated.
 hsModuleForDotProto
     :: MonadError CompileError m
-    => ([HsImportDecl], [HsDecl])
+    => Bool
+    -- ^
```

Comment:
Missing a comment here. Might also be worth a `UnwrapWrapperTypes` newtype or somesuch in order to avoid boolean blindness at call sites.

Reply:
Yes, I agree that wrapping the `Bool` in a newtype is a good idea. Will do this.